apps/assistants/sync.py

```diff
@@ -175,36 +175,43 @@ def is_tool_configured_remotely_but_missing_locally(assistant_data, local_tool_t
 
 
 @wrap_openai_errors
-def are_files_in_sync_with_openai(assistant: OpenAiAssistant) -> bool:
+def are_files_in_sync_with_openai(assistant: OpenAiAssistant) -> tuple[dict[str, list[str]], dict[str, list[str]]]:
```

Comment:
Suggestions to consider to discard
1. Rename the function. The current name suggests a boolean response, but we're returning file diffs
2. Rename `missing_files` to something like `missing_locally` and `extra_files` to `missing_at_openai`.

I would sort of insist on point 1 (about 70%), but point 2 is up to you :smile:

Reply:
👍🏻 ada0ed4befd85fb809835fd75dbeeb90dbc4c6f8